lib/builders/documentTypes/documentTypeBuilder.ts

```diff
@@ -19,7 +19,6 @@ export class DocumentTypeBuilder {
   documentTypeCompositionBuilder: DocumentTypeCompositionBuilder[];
   documentTypeContainerBuilder: DocumentTypeContainerBuilder[];
   id: string;
-  containerId: string;
```

Comment:
It would be nice if you added one more field called 'folder' here to make it consistent with Swagger.